### ISSUE TITLE
Why finetune LLM?

### ISSUE THREAD

**Issue by tian1327** [BatsResearch/ex2#2]
Thanks the authors for the very interesting work! I am wondering why is it necessary to finetune the LLM using the RL based on the reward. I have the following questions.

1. What matters for the classification accuracy is the text descriptions themselves and you can simply use a frozen LLM and ask a bunch of various questions, rather than asking the same question "how to describe {concept}?" And then you can calculate the cosine similarity between different texts and images. To get something similar to those spurious descriptions, you can simply ask "how to describe an online image?" or "what texts are usually associated with an online image?". It seems that those spurious descriptions are mostly some noisy text associated with the online images, e.g. "Zoom in for details".

2. The reward functions encourages the LLM to output descriptions that maximize the similarity between output text and images. But this does not necessarily force LLM to generates descriptions belong to the specific class (because no direct loss back-propagation), as observed like the spurious descriptions. Can RL really discover the most related texts?

Looking forward to hearing from your thoughts! Thank you!

**Comment by Reza-esfandiarpoor** [comment]
Thank you for the great questions.

1. Yes, we can manually engineer prompts to generate descriptions that follow one specific pattern. However, because there is a large number of factors that could impact VLM representations, it would be challenging, if not impossible, to manually engineer prompts to vary every single aspect of descriptions. The RL process allows us to explore a large 
space of potential characteristics more efficiently.

2. We want to generate texts that the VLM deems related to corresponding images. So, although humans might not consider some of the RL-generated descriptions relevant, the VLM sees them as relevant since it assigns them a higher cosine similarity.

**Comment by tian1327** [comment]
Thank you for your prompt answering! 

A follow-up question is, do you think concept names are more important than attributes, i.e. VLMs focus more on names than attributes? 

A recent work of ours REAL find that simply replacing the concept names with its synonyms that are most frequently appeared in VLM's pretraining set yields significant improvement on zero-shot accuracy. It outperforms those methods that add attributes and can even be combined with them to achieve even better performance. You may find it as a both interesting and strong baseline to compare with.

https://shubhamprshr27.github.io/neglected-tails-of-vlms/

Thanks again for the great work and discussions!

**Comment by Reza-esfandiarpoor** [comment]
I think your work provides strong evidence that concept names are very important for VLMs, and we have also observed evidence of this in our experiments!
Congratulations on your paper! Thanks for sharing!
